packages/metadata-fields/BasicMetadata.jsx

```diff
@@ -233,7 +233,7 @@ export default function BasicMetadata({ data }) {
             onClick={copyLink}
             data-role="copy-link"
             type="button"
-            className="su-text-digital-blue su-text-21 su-font-semibold su-mr-auto su-leading-[26.25px]"
+            className="su-text-digital-blue dark:su-text-digital-blue-vivid su-text-21 su-font-semibold su-mr-auto hocus:su-underline su-leading-[125%]"
```

Comment:
Other than the digital-blue-vivid and underline change, I also updated the leading to use `su-leading-[125%]` (we've used this class elsewhere) instead of `su-leading-[26.25px]`.